lib/cogip_defs/Makefile.include

```diff
@@ -1,2 +1,5 @@
 USEMODULE_INCLUDES_cogip_defs := $(LAST_MAKEFILEDIR)/include
 USEMODULE_INCLUDES += $(USEMODULE_INCLUDES_cogip_defs)
+
```

Comment:
Should be in a seperated previous commit

Reply:
Done